### DISCUSSION TITLE
K1/K1C Motherboard Layout

### DISCUSSION THREAD

**Discussion by jaxxtheone** [Guilouz/Creality-Helper-Script-Wiki#567] [Features Request]
Guilouz, I never noticed till today that the K1/K1C motherboard layout has been missing since your last revamp of your Wiki. You may have this missing image and the pinout in your backups. As I'm about to change my motherboard, I did a general search on the Internet but didn't find it. I just found K1 Max's MB. TIA.

**Comment by Guilouz** [comment]
Nothing have changed, board layout are always there : https://guilouz.github.io/Creality-Helper-Script-Wiki/others/boards-layout-k1/

**Reply by jaxxtheone** [reply]
Thank you! Just a remark, this is the K1 Max motherboard. There's no LAN on K1's MB. Also, the hotbed terminals are in different position.
I'm pretty sure you had the two layouts on your late Wiki. I studied carefully the videos from Creality and  noticed the difference.
![FireShot Capture 056 - Creality-Original Kit Mainboard K1 Silencioso, Impressora 3D Parte, P_ - pt aliexpress com](https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/26436482/e17a3d67-eeba-4743-8342-52a18acf1d86)

**Reply by Guilouz** [reply]
No nothing have changed in this page

**Reply by jaxxtheone** [reply]
I just want to point out that the K1 Max motherboard is not the same as the K1/K1C. The one pictured in your Wiki belongs to the K1 Max. The photo above (mine) is from the motherboard of the K1. I searched many sites and didn't find the layout of this motherboard.  Mine doesn't have Ethernet Port neither the Hotbed, which connector is different. If you didn't have it on your Wiki, maybe you can add it as many of us have the K1 printer and need, sooner or later, to service the motherboards of our printers.

**Comment by mguyre** [comment]
Do  you happen to know what port the K1 Max motherboard buzzer (SP1) is connected to?

**Comment by papaguns** [comment]
![IMG_0272](https://github.com/user-attachments/assets/f9e288ab-d0a4-497c-84f3-5e3351a7ebf1)
Does anybody know what are this parts?

**Reply by finnsloss** [reply]
I have exactly the same parts fried on my motherboard. They are devices that are supposed to protect the RS232 serial lines.

The 2 on the top row (in the red circle) are for send and receive on the hotend board. The 2 on the bottom row are for send and receive on the bed board. They link to white and green wires on each cable.

I have error "Key 111 Unknown  Exception" which I believe is a serial communication issue. In my case I think that the power line was shorted to the serial line and blew the components, and unfortunatly I think the serial chip as well. Looks like I will need a new hotend board, and motherboard, and hopefully not the bed board too (though I suspect that might have also blown up).

Looking at RS232 circuit examples, the protection devices can be either zener diodes or thermisters (variable resistors). I think with a code RV they are probably "Resistors Variable". Zener diodes would normall have a D, Z or ZD.

I'll try and test the parts to see if I can identify their value.

**Comment by papaguns** [comment]
![IMG_0273](https://github.com/user-attachments/assets/31199d48-3ce7-49d0-bd25-17c7aa936311)
This details

**Comment by papaguns** [comment]
I tried to change the RS232 on both side(motherboard and hotend board), it
does not help for me. Also I bought new motherboard. And it works.

Сб, 3 мая 2025 г. в 22:34, finnsloss ***@***.***>:

**Reply by twojo** [reply]
I also swapped 3232 ICs on mobo as well as extruder PCB to no avail, after errors then noticing frayed white and green rs232 wires and blown RV25 👎 Waiting to receive 2025 mobo replacement from Creality 🤞
Better extruder power cable management would have prevented this. I'd imagine a lot of people will eventually have this issue. Creality's version of an airbag recall 😅 Maybe why the replacement 2025 mobo was ~$25? Wish it came with an upgraded 4-pin extruder wire harness too
